docs/toc.md

```diff
@@ -1131,12 +1131,12 @@
 <!-- ML.NET Content -->
 # [ML.NET Guide](machine-learning/index.md)
 ## [Tutorials](machine-learning/tutorials/index.md)
-### [Sentiment Analysis (Classification)](machine-learning/tutorials/sentiment-analysis.md)
-### [Taxi Fare Predictor (Regression)](machine-learning/tutorials/taxi-fare.md)
+### [Sentiment analysis (classification)](machine-learning/tutorials/sentiment-analysis.md)
```

Comment:
Can you change this to (binary classification)?

Reply:
@JRAlexander updated!